Yosemite/Yosemite/Actions/NotificationCountAction.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import Storage
+
+// MARK: - NotificationCountAction: Defines all of the Actions supported by the NotificationCountStore.
+//
+public enum NotificationCountAction: Action {
+    /// Increments the notification count by a given amount and type for a site of the given ID.
+    ///
+    case incrementNotificationCount(siteID: Int64, type: Note.Kind, incrementCount: Int, onCompletion: () -> Void)
```

Comment:
I always wonder whether we're being too verbose in our action names. For example, I feel like `NotificationCountAction` gives me enough context to know that this shortened name:

```swift
    case increment(siteID: Int64, type: Note.Kind, change: Int, onCompletion: () -> Void)
```

Will “increment the notification count”. 

🤷

Reply:
I agree, these action names could be more concise since we always call with the enum name. updated in: https://github.com/woocommerce/woocommerce-ios/pull/2320/commits/5d4ac44b89a427313c829dc21062539b2427c8ce